src/main/java/org/gridsuite/modification/server/elasticsearch/EquipmentInfosService.java

```diff
@@ -33,13 +35,19 @@ public class EquipmentInfosService {
     @Value("${spring.data.elasticsearch.partition-size:10000}")
     private int partitionSize;
 
+    public static final Set<String> EXCLUDED_TYPES_FOR_INDEXING = Set.of(IdentifiableType.SWITCH.name());
+
     public EquipmentInfosService(EquipmentInfosRepository equipmentInfosRepository, TombstonedEquipmentInfosRepository tombstonedEquipmentInfosRepository) {
         this.equipmentInfosRepository = equipmentInfosRepository;
         this.tombstonedEquipmentInfosRepository = tombstonedEquipmentInfosRepository;
     }
 
     public void addAllEquipmentInfos(@NonNull final List<EquipmentInfos> equipmentsInfos) {
-        Lists.partition(equipmentsInfos, partitionSize)
+        // get only equipments allowed to be indexed
+        List<EquipmentInfos> filteredEquipmentsInfos = equipmentsInfos.stream()
+                .filter(equipmentInfos -> !EXCLUDED_TYPES_FOR_INDEXING.contains(equipmentInfos.getType()))
+                .toList();
+        Lists.partition(filteredEquipmentsInfos, partitionSize)
                 .parallelStream()
                 .forEach(equipmentInfosRepository::saveAll);
     }
```

Comment:
Add this filter to exclude switches also in addAllTombstonedEquipmentInfos ?
because addAllTombstonedEquipmentInfos is also called in cloneVariantModifications

Reply:
In cloneVariantModifications, we clone the equipment that is already indexed in 'tombstonedEquipment'. However, since we filter the equipment types during the initial indexing, we will not have them here

Reply:
Yes, but what for existing network variants where switches have already been indexed before in tombstonedEquipment ?

Reply:
It will be more easy to just clean elasticsearch indexes, so we will do that :)

Reply:
yes